docs/CONFIG.md

```diff
@@ -14,6 +14,7 @@ The following properties are supported:
 
 | Property | Type | Default | Description |
 |--|--|--|--|
+| `driver_cache_days` | int | 365 | number of days, before the driver manager will question the cache and make a renewal call. |
```

Comment:
IMHO: "...will ask..." or "...will query..." or "...will lookup in the cache..."

Reply:
OK, `invalidate' would suite the meaning, though. No funny phrasing here, I see :P